tiny_dnn/activations/activation_function.h

```diff
@@ -1,241 +0,0 @@
-/*
-    Copyright (c) 2013, Taiga Nomi
-    All rights reserved.
-
-    Use of this source code is governed by a BSD-style license that can be found
-    in the LICENSE file.
-*/
-#pragma once
```

Comment:
@edgarriba There can be a way to keep backward compatibility with old API through indirectly calling new one. One idea is using function templates that prototypes resemble old layer classes constructors. But this may cause overloading ambiguity and we don't have tests for old API anymore, so forfeiting old API looks reasonable decision to me.